main.go

```diff
@@ -262,7 +262,14 @@ func doMain() {
 	logrus.Infoln("using settings: ", spew.Sdump(conf))
 
 	githubClient = clientgithub.NewGitHubClient(conf.githubToken, conf.dryRunMode)
-	r := gin.Default()
+
+	r := gin.New()
+	r.Use(
+		gin.LoggerWithWriter(gin.DefaultWriter, "/_health"),
```

Comment:
Hmm, I'm not completely following, so if I'm running at the `info` level, then what will this show? Will it still ignore `_health`?

Reply:
Yes, the idea is to ignore logging for `_health` endpoint for log levels above `debug` and only have them for `debug` and `trace`.

Reply:
Understood, thx :+1:  I was a bit lazy, but I see now that `_health` is the explicit filter to the logger.
Small suggestion, create a string locally, with `filter := "_health"` and pass it into the `gin.LoggerWithWriter`?

Reply:
Good point! Thanks!